app/presenters/cost_of_living_landing_page_presenter.rb

```diff
@@ -13,4 +13,24 @@ def initialize(content_item)
       end
     end
   end
+
+  def link_clicked_track_data(track_action:, href:)
+    return {} unless internal_link?(href)
+
+    {
+      track_category: "contentsClicked",
+      track_action: track_action,
+      track_label: slug_for_href(href),
+    }
+  end
+
+private
+
+  def internal_link?(link)
+    link.starts_with?("https://www.gov.uk/")
+  end
+
+  def slug_for_href(href)
+    href.gsub("https://www.gov.uk/", "")
```

Comment:
Is this going to cause problems in staging/integration? (I guess we don't really have analytics in those environments, but is it going to cause an unexpected difference in how link attributes are rendered between those and live?)

Reply:
Agree this is sub-optimal. We have a [card](https://trello.com/c/waEkLAut/1279-generate-correct-base-urls-for-internal-links-based-on-environment-m-post-live) in the backlog to generate correct base URLs for internal links based on environment. We'll do that after the page is live.

Reply:
Okay, cool. :)